feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionDetailFragment.kt

```diff
@@ -122,10 +135,10 @@ abstract class SessionDetailFragmentModule {
             return sessionsFragment.viewLifecycleOwner.lifecycle
         }
 
-        @JvmStatic @Provides fun provideActivity(
-            sessionsFragment: SessionDetailFragment
-        ): FragmentActivity {
-            return sessionsFragment.requireActivity()
-        }
+//        @JvmStatic @Provides fun provideActivity(
```

Comment:
If you not need this function, Please delete it 🙏

Reply:
@takahirom 
Sorry, I forgot…
Deleted it.